### ISSUE TITLE
Add Scroll Limit for Ember Mail Retrieval

### ISSUE THREAD

**Issue by balthanon** [Fate-Grand-Automata/FGA#1378]
### Preparation

- [X] No one has asked for this feature before in other issues (even the closed ones)

### Describe the feature

This was partly inspired by the issue fixed in #1340 since it made it very difficult to stop the ember retrieval, so it may not really be necessary at this point, but I think it could be a nice enhancement nonetheless.  

I think it would be nice to add an additional option when you are kicking off ember mail retrieval, either from a lottery screen or from directly in the mailbox, where you can specify the maximum number of times you want FGA to scroll without retrieving any embers before it moves on and retrieves any selected embers.

So basically, if you have 300 ember stacks in your box already and you know that they're all 3 or more, you could set this number to say 5 and if FGA scrolls the screen 5 times without detecting any stacks that it can select, it would just stop there and hit the retrieve button.  As it is, it will scroll dozens of times to reach the bottom of that 300 stacks before stopping.  (Or before #1340 was fixed, it would just scroll infinitely and make it very difficult to actually pause the script.)  This should save a decent amount of time spent scrolling through the mailbox fruitlessly.
